Unsorted "TODO" tasks:

* Fix tests failing with PyPy:

  - sys.getrefcount()
  - test_queues.test_repr
  - test_futures.test_tb_logger_exception_unretrieved

* python3.3 runtests.sh:
  sys:1: ResourceWarning: gc: 3 uncollectable objects at shutdown; use gc.set_debug(gc.DEBUG_UNCOLLECTABLE) to list them
* write unit test for create_connection(ssl=True)
* add COPYING file
* add AUTHORS file and fill author field in setup.py
  (mention Tulip authors)

* Failing tests on Windows, Python 2:

  - test_events.py:ProactorEventLoopTests (SelectorEventLoop is fine)
  - test_windows_events.py
  - test_windows_utils.py (segfault in asyncio.windows_utils.pipe())

* Fix examples:

  - stacks.py: 'exceptions.ZeroDivisionError' object has no attribute '__traceback__'

* as_completed() is a coroutine or not?
* Fix all FIXME in the code
* See:
  http://code.google.com/p/appengine-ndb-experiment/source/browse/ndb/tasklets.py
